src/App.js

```diff
@@ -158,11 +157,6 @@ class OldApp extends Component {
     );
     analyticsV2.track(analyticsV2.event.applicationDidMount);
 
-    /**
-     * This must be saved in the store as early as possible
-     */
-    await saveFCMToken();
```

Comment:
this is redundant as it is handled in the NotificationsHandler which also runs on initialization